file.go

```diff
@@ -23,3 +27,27 @@ func (f File) FilenameWithExtension() string {
 
 	return fmt.Sprintf("%s.%s", text, extension)
 }
+
+// AbsoluteUnixPath returns a fake absolute unix path to a file
+func (f File) AbsoluteUnixPath(nbLen int) string {
+	return filepath.Join(f.Directory(nbLen), f.FilenameWithExtension())
+}
+
+// AbsoluteWin32Path returns a fake absolute win32 path to a file
+func (f File) AbsoluteWin32Path(nbLen int) string {
+	return filepath.Join(f.DriveLetter(), f.Directory(nbLen), f.FilenameWithExtension())
```

Comment:
Can you rename the function to clarify that is a random path to a **file**, renaming it to `AbsoluteFilePathForWindows` and the previous one to `AbsoluteFilePathForUnix`. Please rename `nbLen` to `levels` to clarify that we are talking about how many deep levels the path will have.

One improvement suggestion is to create a `FilePath` function that calls each specific FilePath* function depending where the system is running. So you would have something like:

```go
func (f File) AbsoluteFilePath(levels int) string {
    if runtime.GOOS == "windows" {
        return AbsoluteFilePathForWindows(levels)
    }

    return AbsoluteFilePathForUnix(levels)
}
```

Please also create a test for this function.